src/app.module.ts

```diff
@@ -153,6 +153,7 @@ import { ActivityLogModule } from '@services/api/activity-log/activity.log.modul
                   ...ctx.connectionParams?.headers,
                 },
                 connectionParams: ctx.connectionParams,
+                loaders: dataloaderService.createLoaders(),
```

Comment:
I'd remove the code we added here and add a toDo and link a story where we actually fix this in the toDo